README.md

```diff
@@ -128,26 +131,10 @@ Trails =
 trails:single_host_compile(Trails),
 ```
 
-This way each handler keeps their own routes, as it should be, and you can
+This way each handler maintains their own routes, as it should be, and you can
 merge them easily.
 
 ## Example
 
 For more information about `cowboy_trails`, how to use it and the different
-functions that it exposes, please check this [Example](./example).
-
-## Testing
```

Comment:
Meta-testing no longer requires the PLT to be present from the start. At least I cleared `_build` and ran `rebar3 ct` and didn't see any issues.